### PR TITLE
Customer orders

### PR DESCRIPTION
**Pull request by whenzei**

### CODE REVIEW

File: controllers/orders.js [hunk 1]
```diff
@@ -8,6 +8,23 @@ const psGetFoodStatus = new PS({ name: 'get-food-status', text: `SELECT (dailyLi
...
+const psGetOrders = new PS({
+    name: 'get-orders',
+    text: ` SELECT oid, to_char(orderTime, 'DD-Mon-YYYY HH24:MI') as orderTime, to_char(deliveredTime, 'DD-Mon-YYYY HH24:MI') as deliverTime,
+            streetName, unit, postalCode,
+            case 
+                when (isDeliveryFeeWaived = true) then (finalPrice)
+                else (finalPrice + deliveryFee)
+            end as payablePrice,
+            (SELECT STRING_AGG(fname || ' (x' || qty || ')', ',') FROM Collates where oid = O.oid) foodList,
+            coalesce(P.points, 0) points, coalesce(P.percentOff, 0) discount, P.pid, O.isDeliveryFeeWaived as waived,
```
Comment (jelneo) [inline controllers/orders.js:20]: would be good to standardize whether u want to include the `AS` keyword for column alias

File: controllers/restaurant.js [hunk 2]
```diff
@@ -4,7 +4,29 @@ const PS = require('pg-promise').PreparedStatement;
 const psGetRestaurants = new PS({ name: 'get-retaurants', text: 'SELECT * FROM Restaurants' })
 const psGetMenu = new PS({ name: 'get-menu', text: 'SELECT * FROM Food WHERE rid = $1' });
 const psGetRestaurant = new PS({ name: 'get-restaurant', text: 'SELECT * FROM Restaurants WHERE rid = $1' });
-
+const psGetRestaurantRating = new PS({
+    name: 'get-res-rating',
+    text: `SELECT ROUND(ROUND(AVG(stars)*2, 0) / 2, 1) as stars
```
Comment (jelneo) [inline controllers/restaurant.js]: why is there a need to * 2 then / 2?

File: routes/customer.js [hunk 3]
```diff
@@ -67,7 +69,62 @@ router.get("/account", async (req, res, next) => {
...
+        return next(error);
+    }
+    return res.send(orders);
+})
+
+router.get("/restaurant-rating/:rid", [check('rid').isInt()], validate
+    , async (req, res, next) => {
+        let rating = {};
+        try {
+            console.log(req.params.rid)
```
Comment (jelneo) [inline routes/customer.js]: to remove

File: routes/customer.js [hunk 4]
```diff
@@ -67,7 +69,62 @@ router.get("/account", async (req, res, next) => {
...
+            return next(error);
+        }
+        return res.send(rating);
+    })
+
+router.get("/restaurant-reviews/:rid", [check('rid').isInt()], validate
+    , async (req, res, next) => {
+        let reviews = [];
+        try {
+            console.log(req.params.rid)
```
Comment (jelneo) [inline routes/customer.js]: same as abv

File: controllers/restaurant.js (hunk 2 above)
Comment (whenzei) [reply controllers/restaurant.js]: To round of to the nearest .5 or .0

File: controllers/restaurant.js [hunk 5]
```diff
@@ -2,9 +2,32 @@ const db = require('../db');
 const PS = require('pg-promise').PreparedStatement;
 
 const psGetRestaurants = new PS({ name: 'get-retaurants', text: 'SELECT * FROM Restaurants' })
-const psGetMenu = new PS({ name: 'get-menu', text: 'SELECT * FROM Food WHERE rid = $1 ORDER BY fname' });
```
Comment (jelneo) [inline controllers/restaurant.js:5]: missing `ORDER BY fname`

**Pull request merged**